adding context to FAQ section for redshift serverless (#4247)
segmentio/segment-docs: src/connections/storage/catalog/redshift/index.md

### Can I use an SSH tunnel to connect to my Redshift instance?

Segment does not currently support SSH tunneling to Redshift. You can usually allow Segment's ETL to write to Redshift without leaving the cluster available to other connections by using IP level restrictions.

### Do you support Redshift Serverless?

> Segment does not currently support Serverless Redshift. While you are able to set up the connection in the Segment UI, we do not have the functionality to query Redshift's SYS tables.

Reviewer: We try to avoid first person pronouns (for example 'we', 'I', 'us', 'ours') in the docs whenever possible, because it can create confusion!

```suggestion
Segment does not currently support Serverless Redshift. While you can set up the connection in the Segment app, Segment does not have the functionality to query Redshift's SYS tables.
```
